cmake/CmakeFunctionHelper.cmake

```
@@ -244,14 +244,24 @@ endfunction(generate_blas_binary_special_objects)
 # blas ternary function for generating source code
 function(generate_blas_ternary_objects blas_level func)
 set(LOCATION "${SYCLBLAS_GENERATED_SRC}/${blas_level}/${func}/")
+string(FIND ${func} "_const" pos)
```

```suggestion
string(FIND ${func} "_const" const_pos)
```
Or something like that, to make it more descriptive